coordinator/src/main/scala/filodb.coordinator/client/QueryCommands.scala

```diff
@@ -35,19 +35,40 @@ object QueryCommands {
 
   final case class SpreadChange(time: Long = 0L, spread: Int = 1)
 
+  trait SpreadProvider {
+    def spreadFunc(filter: Seq[ColumnFilter]): scala.Seq[SpreadChange]
```

Comment:
Please remove the `scala.` prefix.  That is annoying IntelliJ auto insert stuff which is not necessary.

Reply:
fixed in 5843940